dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskExecutionContext.java

```diff
@@ -126,6 +126,11 @@ public class TaskExecutionContext implements Serializable {
 
     private boolean failover;
 
+    /**
+     * Timestamp (ms) when the task was first enqueued for dispatch.
+     */
+    private final long firstDispatchEnqueueTimeMs = System.currentTimeMillis();
```

Comment:
```suggestion
    private final long firstDispatchTime = System.currentTimeMillis();
```
This is more like the creation time of the `taskExecutionContext`.

Reply:
> This is more like the creation time of the `taskExecutionContext`.

ok